.github/workflows/stylebot.yaml

```
@@ -0,0 +1,67 @@
+          app-id: ${{ secrets.STYLEBOT_GITHUB_APP_ID }}
+          private-key: ${{ secrets.STYLEBOT_GITHUB_APP_KEY }}
+      - uses: actions/checkout@master
+      - id: nodeversion
+        run: echo "version=$(grep nodejs .tool-versions | sed -e 's/[^[:space:]]*[[:space:]]*//')" >> $GITHUB_OUTPUT
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ steps.nodeversion.outputs.version }}
+          cache: yarn
+      - run: yarn install --immutable
+      # Run fixes, save stdout.
+      - run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:31: Double quote to prevent globbing and word splitting

(shellcheck)